js/admin.vue

```diff
@@ -1,9 +1,11 @@
 <template>
+<div>
```

Comment:
I tried not to indent anymore when it's possible to limit the diff size.
I can do it but it will produce a very very big diff.